<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftRegisters</class>
 <widget class="QWidget" name="ShiftRegisters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1,1,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0">
      <item row="1" column="1">
       <widget class="QLabel" name="label_LatchPin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>NotDefined</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="text_latchPin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Latch pin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="text_shiftRegisterType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Shift register type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="text_dataPin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Data pin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="text_registersCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Registers count</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_ShiftIndex">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="spinBox_ButtonCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="CenteredCBox" name="comboBox_ShiftRegType">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_RegistersCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_DataPin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>NotDefined</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" colspan="3">
       <widget class="QLabel" name="text_buttonCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Button count</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CenteredCBox</class>
   <extends>QComboBox</extends>
   <header>centered_cbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
